### PR TITLE
Added log forwarding collector

### PR DESCRIPTION
**Pull request by christopherhans**
Testing if vCSA is sending logs to configured log hosts.

### CODE REVIEW

File: collectors/LoggingCollector.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    def __init__(self, vcenter):
+        self.vcenter = vcenter
+        self.connection_states = {
+            'UP': 1,
+            'UNKNOWN': 2,
+            'DOWN': 0
+        }
+
+    def describe(self):
+        yield GaugeMetricFamily('vcsa_logging_status', 'Checks the log forwarding of vCSA.')
```
Comment (viennaa) [inline collectors/LoggingCollector.py:16]: Please write short info about status in help message, like: `Down: 0 Up: 1 ..`

File: modules/Connection.py [hunk 2]
```diff
@@ -38,7 +38,24 @@ def get_request(target, key, session_id):
...
+        except Exception as e:
+            print("Problem handling", key, "for", target, ":", str(e))
+            return False
+        if response.status_code == 200:
+            return response.json()
+        else:
+            print("Problem with post return of", target, ":", response.text)
+            return False
+
+        # going to be used until reuse of session_id is in place
```
Comment (viennaa) [inline modules/Connection.py:58]: fix indent

File: modules/Connection.py [hunk 3]
```diff
@@ -38,7 +38,24 @@ def get_request(target, key, session_id):
             print("Problem with get return of", target, ":", response.text)
             return False
 
-    # going to be used until reuse of session_id is in place
+    def post_request(target, key, post, session_id):
```
Comment (viennaa) [inline modules/Connection.py:41]: I'd rename `post` to `data`

**Review by viennaa — CHANGES_REQUESTED** [review]
Looks pretty much like the collector in place already ;D.
Nice improvement would be, if you could introduce real python logging and get rid of prints.
Some minor remarks are in code.

**Comment by viennaa** [comment]
Additionally, please change Readme, it has more than one Collector now.

File: modules/Connection.py (hunk 3 above)
Comment (christopherhans) [reply modules/Connection.py:41]: But isn't it a post? Named at similar to the `get_request`

Comment (viennaa) [reply modules/Connection.py:41]: I mean in the arguments. Otherwise it reads a little odd imho: `foo.post_request(..,.., post,..)`

Comment (christopherhans) [reply modules/Connection.py:41]: Ah, gotcha

**Comment by christopherhans** [comment]
> Looks pretty much like the collector in place already ;D.
> Nice improvement would be, if you could introduce real python logging and get rid of prints.
> Some minor remarks are in code.

Need to get this live first --> Audit gap.

But yes, Python logging would definitely be an improvement.

**Review by viennaa — APPROVED** [review]

**Pull request merged**